static/app/views/replays/detail/memoryChart.tsx

```diff
@@ -16,30 +20,37 @@ type Props = {
 const formatTimestamp = timestamp =>
   getFormattedDate(timestamp * 1000, 'MMM D, YYYY HH:mm:ss');
 
+const getRoundedMemorySizeFromBytes = bytes => {
+  const convertedValue = formatBytesBase2(bytes);
+  const values = convertedValue.split(' ');
+  // we round the byte size to the nearest ten and put the string back together
+  return `${Math.round(Number(values?.[0] || 0) / 10) * 10} ${values[1]}`;
+};
+
 function MemoryChart({memorySpans = []}: Props) {
+  const theme = useTheme();
   if (memorySpans.length <= 0) {
     return <EmptyMessage>{t('No memory metrics exist for replay.')}</EmptyMessage>;
   }
 
   const chartOptions: Omit<AreaChartProps, 'series'> = {
-    grid: {
-      left: '10px',
-      right: '10px',
+    grid: Grid({
       top: '40px',
-      bottom: '10px',
-    },
-    tooltip: {
+    }),
+    tooltip: Tooltip({
       trigger: 'axis',
       valueFormatter: (value: number | null) => formatBytesBase2(value || 0),
-    },
-    xAxis: {
+    }),
+    xAxis: XAxis({
       type: 'category',
       min: formatTimestamp(memorySpans[0]?.timestamp),
```

Comment:
The docs show that `'time'` is a valid value: https://echarts.apache.org/en/option.html#xAxis.type
idk if this is the same version that we have installed though

```suggestion
    xAxis: XAxis({
      type: 'time',
      min: formatTimestamp(memorySpans[0]?.timestamp),
```

Reply:
I didn't have any trouble when all the values were numbers but when I format the timestamps the chart doesn't want to render the data if it set it to 'time'. I could try to dig into more if you'd like.

Reply:
Could it be that the `min` and `max` should be numbers when type=time?

Sorry, i hadn't noticed that `'time'` was the thing that made break last time. Probably not worth spending all the time on it now.

Reply:
All good, I tried time without formatting the min and max and it still didn't show. 

```
    xAxis: XAxis({
      type: 'time',
      min: memorySpans[0]?.timestamp,
      max: memorySpans[memorySpans.length - 1]?.timestamp,
      axisLabel: {
        formatter: value => formatTimestamp(value),
      },
      theme,
    }),
```

Reply:
Though if I didn't format the name the x-axis ticks did show up. I'm not entirely sure what echarts wants from me to make it all place nicelol